src/Xamarin.Android.Build.Tasks/Xamarin.Android.Application.targets

```diff
@@ -35,4 +35,29 @@ Copyright (C) 2019 Microsoft Corporation. All rights reserved.
     </GetAndroidPackageName>
     <Exec Command="&quot;$(AdbToolPath)adb&quot; $(AdbTarget) shell am force-stop &quot;$(_AndroidPackage)&quot;" />
   </Target>
+  <PropertyGroup>
+    <AndroidAotCustomProfile Condition=" '$(AndroidAotCustomProfile)' == '' ">custom.aprof</AndroidAotCustomProfile>
+    <AndroidAotProfilerPort Condition=" '$(AndroidAotProfilerPort)' == '' ">9999</AndroidAotProfilerPort>
```

Comment:
This likewise should be documented in the Properties section of `BuildProcess.md`.

Reply:
It is really specific to the aot profiling targets, so I am not sure it makes sense to add it to properties sections. It is same case as $(AndroidLaunchActivity), which is also only documented in the build targets section.